rfc6962-bis.xml

```diff
@@ -1153,10 +1153,13 @@ entry specified by <spanx style="verb">start</spanx>.
     </section>
     <section title="TLS Servers" anchor="tls_servers">
       <t>
-        TLS servers MUST use at least one of the three mechanisms listed below to present one or more SCTs or inclusion proofs from one or more logs to each TLS client during TLS handshakes, where each SCT or inclusion proof corresponds to the server certificate or to a name-constrained intermediate the server certificate chains to. Three mechanisms are provided because they have different tradeoffs.
+        TLS servers MUST use at least one of the three mechanisms listed below to present one or more SCTs from one or more logs to each TLS client during TLS handshakes, where each SCT corresponds to the server certificate or to a name-constrained intermediate the server certificate chains to. TLS servers MAY present relevant inclusion proofs together with Signed Tree Heads(<spanx style="verb">TransItem</spanx> of types <spanx style="verb">inclusion_proof</spanx> or <spanx style="verb">signed_tree_head</spanx>) in addition to the SCTs. When presented, the inclusion proof SHOULD be for computing the root in the accompanying Signed Tree Head.
```

Comment:
Yes, that's a good idea ... reducing complexity in the client always seems like a good plan.

Eran wants clients to be able to only check the SCT, but that would still be possible with a dedicated data structure.